### PR TITLE
fix(registration-api): #2599 fix endpoint permission requirements

### PR DESCRIPTION
**Pull request by markkopaas**

### CODE REVIEW

**Review by oyvindis — APPROVED** [review]

File: applications/registration-api/src/main/java/no/dcat/configuration/FdkPermissionEvaluator.java [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 
 import java.io.Serializable;
 
+@Component
 public class FdkPermissionEvaluator implements PermissionEvaluator {
-    private static final Logger logger = LoggerFactory.getLogger(FdkPermissionEvaluator.class);
 
+    private PermissionsService permissionsService;
+
+    public FdkPermissionEvaluator(PermissionsService permissionsService) {
```
Comment (azaxarov) [inline applications/registration-api/src/main/java/no/dcat/configuration/FdkPermissionEvaluator.java]: Lombok constructor?

File: applications/registration-api/src/main/java/no/dcat/controller/CatalogController.java [hunk 2]
```diff
@@ -55,9 +56,9 @@
 
 
     @Autowired
-    public CatalogController(CatalogRepository catalogRepository, SpringSecurityContextService springSecurityContextService, HarvesterService harvesterService, EnhetService enhetService) {
+    public CatalogController(CatalogRepository catalogRepository, PermissionsService permissionsService, HarvesterService harvesterService, EnhetService enhetService) {
```
Comment (azaxarov) [inline applications/registration-api/src/main/java/no/dcat/controller/CatalogController.java]: Lombook constructor

File: applications/registration-react/src/config.js [hunk 3]
```diff
@@ -70,5 +70,7 @@ export const loadConfig = async () => {
   //   SSO_HOST: 'http://localhost:8084'
   // });
 
+  // use local server on host.
+  // env.REGISTRATION_API_HOST = 'http://localhost:8115';
```
Comment (azaxarov) [inline applications/registration-react/src/config.js]: Is this related to this PR?

**Review by azaxarov — CHANGES_REQUESTED** [review]

File: applications/registration-api/src/main/java/no/dcat/configuration/FdkPermissionEvaluator.java (hunk 1 above)
Comment (markkopaas) [reply applications/registration-api/src/main/java/no/dcat/configuration/FdkPermissionEvaluator.java]: fikset

File: applications/registration-react/src/config.js (hunk 3 above)
Comment (markkopaas) [reply applications/registration-react/src/config.js]: rebased

File: applications/registration-api/src/main/java/no/dcat/controller/CatalogController.java (hunk 2 above)
Comment (markkopaas) [reply applications/registration-api/src/main/java/no/dcat/controller/CatalogController.java]: rebased

File: applications/registration-api/src/main/java/no/dcat/configuration/FdkPermissionEvaluator.java (hunk 1 above)
Comment (markkopaas) [reply applications/registration-api/src/main/java/no/dcat/configuration/FdkPermissionEvaluator.java]: rebased

**Pull request merged**